examples/ucnf-kiknos/scripts/deploy_kiknos.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+NSE_ORG=${NSE_ORG:-mmatache}
+NSE_TAG=${NSE_TAG:-kiknos}
+PULL_POLICY=${PULL_POLICY:-IfNotPresent}
+SERVICE_NAME=${SERVICE_NAME:-hello-world}
+DELETE=${DELETE:-false}
+OPERATION=${OPERATION:-apply}
+SUBNET_IP=${SUBNET_IP:-192.168.254.0}
+
+function print_usage() {
+    echo "$(basename "$0") - Deploy NSM Kiknos topology. All properties can also be provided through env variables
+
+NOTE: The defaults will change to the env values for the ones set.
+
+Usage: $(basename "$0") [options...]
+Options:
+  --cluster             Cluster name                                                        env var: CLUSTER          - (Default: $CLUSTER)
+  --cluster-ref         Cluster reference                                                   env var: CLUSTER_REF      - (Default: $CLUSTER_REF)
```

Comment:
is cluster-ref the remote cluster's context?

Reply:
right